Editor/EditorCore/EditorShapeUtility.cs

```diff
@@ -84,29 +100,40 @@ public static List<GUIContent[]> shapeTypesGUI
 
 
         static EditorShapeUtility()
+        {
+            ResetPrefs();
```

Comment:
why are we resetting the preferences in constructor?

Reply:
Reset Prefs is also in charge of Initializing the data structure storing the preferences if not done yet